.buildkite/scripts/integration_tests.sh

```diff
@@ -21,7 +21,7 @@ fi
 make mage
 
 # PACKAGE
-DEV=true EXTERNAL=true SNAPSHOT=true PLATFORMS=linux/amd64,linux/arm64 PACKAGES=tar.gz mage package
+DEV=true EXTERNAL=true SNAPSHOT=true PLATFORMS=linux/amd64,linux/arm64,windows/amd64 PACKAGES=tar.gz,zip mage package
```

Comment:
Now that the list of platforms and packages has grown a bit, I wonder if we should provide a `mage integration:package` (or similar) target for developers to easily run this command locally.  Not necessarily for this PR; could be done in a separate PR.